demo-shell/src/app/components/search/search-bar.component.ts

```diff
@@ -27,23 +26,7 @@ import { AuthenticationService } from '@alfresco/adf-core';
 })
 export class SearchBarComponent {
 
-    @Input()
-    expandable: boolean = true;
-
-    @Output()
-    expand = new EventEmitter();
```

Comment:
also expand

Reply:
not used in components, just an old demo shell code